types/knockout/index.d.ts

```diff
@@ -151,12 +152,6 @@ interface KnockoutAllBindingsAccessor {
 }
 
 interface KnockoutBindingHandler {
-    after?: Array<string>;
```

Comment:
Why did these lines get removed? Could you point to the documentation where this changes outlined.

Reply:
I cannot point to documentation.  I specifically removed these lines because they cause TypeScript compiler errors.  Specifically type mismatch errors.  This is the only way I found, short of just replacing the Knockout type definition with a wildcard (and thus eliminating the benefit of TypeScript type checking) for Knockout to be used with TypeScript.  Either this definition without my changes outright does NOT work with the latest versions of TypeScript or there are some compiler settings I need to change that I am not aware of.

Reply:
Would you mind sharing the errors you were seeing? By removing all keys in this interface, you make this type equivalent to `{}` which is a nearly useless type. Perhaps there is a lighter approach than deleting all of the types to fix the errors you were seeing?